site/src/pages/HealthPage/HealthPage.tsx

```diff
@@ -0,0 +1,242 @@
+import Box from "@mui/material/Box"
+import { useQuery } from "@tanstack/react-query"
+import { getHealth } from "api/api"
+import { Loader } from "components/Loader/Loader"
+import { useTab } from "hooks"
+import { Helmet } from "react-helmet-async"
+import { pageTitle } from "utils/page"
+import { colors } from "theme/colors"
+import CheckCircleOutlined from "@mui/icons-material/CheckCircleOutlined"
+import ErrorOutline from "@mui/icons-material/ErrorOutline"
+import { SyntaxHighlighter } from "components/SyntaxHighlighter/SyntaxHighlighter"
+import { Stack } from "components/Stack/Stack"
+import {
+  FullWidthPageHeader,
+  PageHeaderTitle,
+  PageHeaderSubtitle,
+} from "components/PageHeader/FullWidthPageHeader"
+import { Stats, StatsItem } from "components/Stats/Stats"
+import { makeStyles } from "@mui/styles"
+
+const sections = {
+  derp: "DERP",
+  access_url: "Access URL",
+  websocket: "Websocket",
+  database: "Database",
+} as const
+
+export default function HealthPage() {
+  const tab = useTab("tab", "derp")
+  const { data: healthStatus } = useQuery({
+    queryKey: ["health"],
+    queryFn: () => getHealth(),
+  })
+
+  return (
+    <>
+      <Helmet>
+        <title>{pageTitle("Health")}</title>
+      </Helmet>
+
+      {healthStatus ? (
+        <HealthPageView healthStatus={healthStatus.data} tab={tab} />
+      ) : (
+        <Loader />
+      )}
+    </>
+  )
+}
+
+export function HealthPageView({
+  healthStatus,
+  tab,
+}: {
+  healthStatus: Awaited<ReturnType<typeof getHealth>>["data"]
+  tab: ReturnType<typeof useTab>
+}) {
+  const styles = useStyles()
+
+  return (
+    <Box
+      sx={{
+        height: "calc(100vh - 62px - 36px)",
+        overflow: "hidden",
+        // Remove padding added from dashboard layout (.siteContent)
+        marginBottom: "-48px",
+      }}
+    >
+      <FullWidthPageHeader sticky={false}>
+        <Stack direction="row" spacing={2} alignItems="center">
+          {healthStatus.healthy ? (
+            <CheckCircleOutlined
+              sx={{
+                width: 32,
+                height: 32,
+                color: (theme) => theme.palette.success.light,
+              }}
+            />
+          ) : (
+            <ErrorOutline
+              sx={{
+                width: 32,
+                height: 32,
+                color: (theme) => theme.palette.error.main,
+              }}
+            />
+          )}
+
+          <div>
+            <PageHeaderTitle>
+              {healthStatus.healthy ? "Healthy" : "Unhealthy"}
+            </PageHeaderTitle>
+            <PageHeaderSubtitle>
+              {healthStatus.healthy
+                ? "Your deploy is working as expected"
+                : "Check the errors below"}
+            </PageHeaderSubtitle>
+          </div>
+        </Stack>
+
+        <Stats aria-label="Deployment details" className={styles.stats}>
+          <StatsItem
+            className={styles.statsItem}
+            label="Last check"
```

Comment:
Will the page update automatically is something bad is happening? Let's say I'd like to have the Health page open and be sure that it refreshes.

Reply:
It will not, but it is a good feature. Should we pull the info every 5 seconds? cc.: @bpmct